tests/cli/commands/remote_commands/test_config_command.py

```diff
@@ -232,3 +235,182 @@ def test_should_raise_exception_when_option_is_missing(self, caplog):
             self.parser.parse_args(["config", "get-value", "missing-section", "dags_folder"])
         )
         assert "section/key [missing-section/dags_folder] not found in config" in caplog.text
+
+
+class TestConfigLint:
+    @pytest.mark.parametrize("removed_config", config_command.CONFIGS_CHANGES)
+    def test_lint_detects_removed_configs(self, removed_config):
+        with mock.patch("airflow.configuration.conf.has_option", return_value=True):
+            with contextlib.redirect_stdout(StringIO()) as temp_stdout:
+                config_command.lint_config(cli_parser.get_parser().parse_args(["config", "lint"]))
+
+            output = temp_stdout.getvalue()
+
+        normalized_output = re.sub(r"\s+", " ", output.strip())
+        normalized_message = re.sub(r"\s+", " ", removed_config.message.strip())
+
+        assert normalized_message in normalized_output
+
+    @pytest.mark.parametrize(
+        "section, option, suggestion",
+        [
+            (
+                "core",
+                "check_slas",
+                "The SLA feature is removed in Airflow 3.0, to be replaced with Airflow Alerts in future",
+            ),
+            (
+                "core",
+                "strict_asset_uri_validation",
+                "Asset URI with a defined scheme will now always be validated strictly, raising a hard error on validation failure.",
+            ),
+            (
+                "logging",
+                "enable_task_context_logger",
+                "Remove TaskContextLogger: Replaced by the Log table for better handling of task log messages outside the execution context.",
+            ),
+        ],
+    )
+    def test_lint_with_specific_removed_configs(self, section, option, suggestion):
+        with mock.patch("airflow.configuration.conf.has_option", return_value=True):
+            with contextlib.redirect_stdout(StringIO()) as temp_stdout:
+                config_command.lint_config(cli_parser.get_parser().parse_args(["config", "lint"]))
+
+            output = temp_stdout.getvalue()
+
+        normalized_output = re.sub(r"\s+", " ", output.strip())
+
+        expected_message = f"Removed deprecated `{option}` configuration parameter from `{section}` section."
+        assert expected_message in normalized_output
+
+        if suggestion:
```

Comment:
It seems the suggestion always exists. I think we can remove this `if` black

Reply:
Changed it